cmd/tar2rpm/main.go

```diff
@@ -102,11 +102,15 @@ func main() {
 		fmt.Fprintln(os.Stderr, "reading tar content from stdin.")
 		i = os.Stdin
 	case 1:
-		f, err := os.Open(flag.Arg(0))
-		if err != nil {
-			log.Fatalf("Failed to open file %s for reading\n", flag.Arg(0))
+		if flag.Arg(0) == "-" {
```

Comment:
Years from now, someone is going to be surprised by the difference between this and "case 0", and they will try to fix this.

Can you add a comment, explaining the rationale?

Reply:
Done

Reply:
I don't see the change?